### ISSUE TITLE
EPIC: As a blind or low vision IMAGE user, I want to know about human action information in photographs, so that I can better understand what the people in a photograph are currently doing.

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#619]
@rbazin suggests this breakdown (would need to be converted into sprint stories)

Hi, here is what would be my approach for action recognition, turned into a tentative schedule for the summer intern :
3 months = 12 weeks schedule
Look for available models (image classification) and datasets (action recognition on images) (2 weeks)
If no convincing datasets found, build a dataset of images from [datasets of videos](https://paperswithcode.com/task/action-recognition-in-videos) (3 weeks)
Gather datasets of videos for action recognition
Extract frames from the videos using pre-trained available models, based on intensity ?
Verify the dataset manually and correct it if needed
Train a [classification model](https://github.com/open-mmlab/mmclassification) in a supervised manner on the dataset built (3 weeks)
Test its performance on validation set AND test set from IMAGE-test-graphics (1 week)
Turn the standalone model into a pre-processor (2 weeks)
Test pre-processor on unicorn (1 week)
Document research, choices, tests and possible future work (THROUGHOUT)
Feel free to comment or change anything, hope it will help.

RESOURCES
@cyan Human Action Recognition dataset
https://www.kaggle.com/datasets/meetnagadia/human-action-recognition-har-dataset

@emmanuelwilson  http://human-pose.mpi-inf.mpg.de/

@VenissaCarolQuadros https://github.com/OlafenwaMoses/Action-Net

@rohanakut 
There also might be an opportunity to use [caption recignition datasets](https://www.kaggle.com/datasets/hsankesara/flickr-image-dataset) for extracting the action. I think that might be more suitable compared to a video based dataset. We could use POS tagging to extract the verb from the caption and that would be our action
